package.json

```diff
@@ -10,8 +10,19 @@
     "underscore": "~1.8.3"
   },
   "devDependencies": {
-    "jshint": "^2.7.0",
     "edx-custom-a11y-rules": "edx/edx-custom-a11y-rules",
+    "jasmine-core": "^2.4.1",
+    "jasmine-jquery": "^2.1.1",
+    "jquery": "^2.1.4",
+    "jshint": "^2.7.0",
+    "karma": "^0.13.22",
+    "karma-chrome-launcher": "^0.2.3",
+    "karma-coverage": "^0.5.5",
+    "karma-firefox-launcher": "^0.1.7",
+    "karma-jasmine": "^0.3.8",
+    "karma-jasmine-html-reporter": "^0.2.0",
+    "karma-junit-reporter": "^0.4.1",
+    "karma-requirejs": "^0.2.6",
```

Comment:
Please add karma-spec-reporter too. We've found it really useful in the UI Toolkit:

https://github.com/edx/edx-ui-toolkit/pull/10